examples/sunpy_other_packages/reprojection_aia_euvi_mosaic.py

```diff
@@ -0,0 +1,220 @@
+"""
+=========================================
+Creating a Full Sun Map with AIA and EUVI
+=========================================
+
+In this example we will combine three EUV images, one from AIA and one
+from each of EUVI_A and EUVI_B into a full latitude / longitude map of
+the Sun.
+
+"""
+# sphinx_gallery_thumbnail_number = 4
+
+import matplotlib.pyplot as plt
+import numpy as np
+from reproject import reproject_interp
+from reproject.mosaicking import reproject_and_coadd
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+from astropy.wcs import WCS
+
+import sunpy.map
+import sunpy.sun
+from sunpy.coordinates import get_body_heliographic_stonyhurst
+from sunpy.net import Fido
+from sunpy.net import attrs as a
+
+######################################################################
+# To get started let’s download the data:
+#
+
+stereo = (a.Instrument('EUVI') &
+          a.Time('2011-11-01', '2011-11-01T00:10:00'))
+
+aia = (a.Instrument('AIA') &
+       a.vso.Sample(24 * u.hour) &
+       a.Time('2011-11-01', '2011-11-02'))
+
+wave = a.Wavelength(19.5 * u.nm, 19.5 * u.nm)
+
+res = Fido.search(wave, aia | stereo)
+
+files = Fido.fetch(res)
+
+
+######################################################################
+# Next we create a SunPy map for each of the files.
+#
+maps = sunpy.map.Map(sorted(files))
+
+
+######################################################################
+# When combining these images all three need to assume the same radius of
+# the Sun for the data. The AIA images specify a slightly different value
+# than the IAU 2015 constant. To avoid coordinate transformation issues we
+# reset this here.
+#
+
+maps[0].meta['rsun_ref'] = sunpy.sun.constants.radius.to_value(u.m)
+
+
+######################################################################
+# Next we will plot the locations of the three spacecraft with respect to
+# the Sun so we can easily see the relative separations.
+#
+
+earth = get_body_heliographic_stonyhurst('earth', maps[0].date)
+
+plt.figure(figsize=(8, 8))
+r_unit = u.AU
+
+ax = plt.subplot(projection='polar')
+circle = plt.Circle((0.0, 0.0), (10*u.Rsun).to_value(r_unit),
+                    transform=ax.transProjectionAffine + ax.transAxes, color="yellow",
```

Comment:
This is a bit complicated, why does it need the transform bit?

Reply:
I copied this out of @hayesla 's other example of plotting all the planet positions, so I am not 100% sure!